### PR TITLE
fix(all): helper function return false on ID of 0

### PR DESCRIPTION
**Pull request by griest024**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

The helper functions would return `false` when the factory generated a field with `0`.

## What is the new behavior?
All `faker.random.number` calls have a minimum value of `1`, which will evaluate to `true` in the helper functions.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

**Comment by griest024** [comment]
@lderrickable I went ahead and changed every single random number generation to have a min value of 1. Can you make sure that won't break anything, especially in category and product?

**Comment by griest024** [comment]
Would it be better to change the helper functions instead?

File: libs/checkout/testing/src/order/factories/order-payment.factory.ts [hunk 1]
```diff
@@ -9,15 +9,15 @@ import { DaffModelFactory } from '@daffodil/core/testing';
...
+    quote_id = faker.random.number({min: 1, max: 1000});
     created_at = new Date();
     updated_at = new Date();
     method = 'card';
     cc_type = 'visa';
-    cc_number_enc = faker.random.number(1000).toString();
-    cc_last4 = faker.random.number(1000).toString();
-    cc_cid_enc = faker.random.number(1000).toString();
+    cc_number_enc = faker.random.number({min: 1, max: 1000}).toString();
+    cc_last4 = faker.random.number({min: 1, max: 1000}).toString();
```
Comment (lderrickable) [inline libs/checkout/testing/src/order/factories/order-payment.factory.ts:19]: this should be `{min: 1000, max: 9999}`

File: libs/order/testing/src/factories/order-payment.factory.ts [hunk 2]
```diff
@@ -6,13 +6,13 @@ import * as faker from 'faker/locale/en_US';
...
-    payment_id = faker.random.number(1000);
-    order_id = faker.random.number(1000);
+    payment_id = faker.random.number({min: 1, max: 1000});
+    order_id = faker.random.number({min: 1, max: 1000});
     created_at = faker.date.past().toString();
     updated_at = faker.date.past().toString();
     method = 'card';
     cc_type = 'visa';
-    cc_last4 = faker.random.number(1000).toString();
+    cc_last4 = faker.random.number({min: 1, max: 1000}).toString();
```
Comment (lderrickable) [inline libs/order/testing/src/factories/order-payment.factory.ts:15]: same here

File: libs/checkout/testing/src/order/factories/order-payment.factory.ts (hunk 1 above)
Comment (griest024) [reply libs/checkout/testing/src/order/factories/order-payment.factory.ts:19]: done

File: libs/order/testing/src/factories/order-payment.factory.ts (hunk 2 above)
Comment (griest024) [reply libs/order/testing/src/factories/order-payment.factory.ts]: done

**Pull request merged**